sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1445,6 +1445,40 @@ case class ToUTCTimestamp(left: Expression, right: Expression) extends UTCTimest
   override val prettyName: String = "to_utc_timestamp"
 }
 
+
+
+case class AddHours(startTime: Expression, numHours: Expression, timeZoneId: Option[String] = None)
+  extends BinaryExpression with ImplicitCastInputTypes with NullIntolerant
+    with TimeZoneAwareExpression {
```

Comment:
I read the scala-style-guide and wasn't able to decipher the best practice for this specific scenario.  I opened [a scala-style-guide issue to clarify](https://github.com/databricks/scala-style-guide/issues/81).  Do you think this is better?

```scala
  extends BinaryExpression with ImplicitCastInputTypes with NullIntolerant
  with TimeZoneAwareExpression {
```

I'm glad you made the comment because it's really important to manage whitespace consistently throughout the codebase.